frontend/workflows/experimentation/src/index.tsx

```diff
@@ -33,5 +33,5 @@ const register = (): WorkflowConfiguration => {
 };
 
 export default register;
-
+export type { FormItem } from "./core/form-fields";
```

Comment:
This [line](https://github.com/lyft/clutch/blob/main/frontend/workflows/serverexperimentation/src/start-experiment.tsx#L19) throws an error for relative imports, so I exported the type instead, which is the preferred method anyways.

> Relative import from another package is not allowed. Use `@clutch-sh/experimentation/src/core/form-fields` instead of `../../experimentation/src/core/form-fields`eslint[import/no-relative-packages](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-relative-packages.md)